packages/next-swc/crates/next-core/src/app_segment_config.rs

```diff
@@ -55,7 +55,7 @@ pub enum NextSegmentFetchCache {
 pub struct NextSegmentConfig {
     pub dynamic: NextSegmentDynamic,
     pub dynamic_params: bool,
-    pub revalidate: bool,
+    pub revalidate: u32,
```

Comment:
unfortunately it's `false | 'force-cache' | 0 | number`

https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config#revalidate

Reply:
updated